### DISCUSSION TITLE
Changed vendor username

### DISCUSSION THREAD

**Discussion by tshakalekholoane** [github-linguist/linguist#5562] [Q&A] [answered]
Do I need to submit a pull request to reflect changes in my username?

I contributed the syntax highlighting grammar for Hy files https://github.com/leveson/vscode-hy and for now it seems like GitHub is handling the redirects automatically but I am not sure for how much longer and how and if that affects Git submodules as in this case.

**Comment by lildude** [comment, marked as answer]
Yes, please submit a PR to update the submodule URL. Whilst GitHub should retain the redirect indefinitely, it's best to be safe and not rely on it.
